frontend/config/plugins.js

```diff
@@ -25,16 +25,6 @@ const plugins = [
   new CopyWebpackPlugin({
     patterns: [{ from: 'static' }],
   }),
-  new DefinePlugin({
-    K8S_TARGET_URL: JSON.stringify(
```

Comment:
@karelhala if these are no longer used, can we remove the `globals` entries for them in `.eslintrc.js` and `index.d.ts`?

Reply:
Also wanted to confirm we did not need any additional changes on Akamai before merging this PR.

Reply:
Mhh, good point. We are not using WS when deployed, however we should sync this with Akamai change to add the proxy in there to have the k8s ws test available.